index.html

```diff
@@ -35,8 +34,7 @@
           </ul>
         </nav>
       </header>
-    </section>
-    <section class="hero-heading hero">
+       <section class="hero-heading hero">
       <h1 class="title">Hey there, I am Dennis.</h1>
```

Comment:
- [ ] I think there is an issue with your headings because you are currently using `<h1>` 9 times and there should only be 1 `<h1>` tag per page. I think it's ok that you use the `<h1>` tag for this headline title because it's the most important (semantically speaking :smiley:), and you could use the `<h2>` tag for the section titles and the `<h3>` for less important headings like project titles :wink: :pushpin: